### PR TITLE
fix: adding back in setTargetResolution

### PR DESCRIPTION
**Pull request by bswhite1**
- Fixes the scan window on Samsung A23. Also verified it works on Moto G Play, Moto G Power (2021), and Pixel 6.
- Pull request for https://github.com/juliansteenbakker/mobile_scanner/issues/604

### CODE REVIEW

**Comment by bswhite1** [comment]
Pull request for https://github.com/juliansteenbakker/mobile_scanner/issues/604

File: android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt [hunk 1]
```diff
@@ -186,7 +188,12 @@ class MobileScanner(
             // Build the analyzer to be passed on to MLKit
             val analysisBuilder = ImageAnalysis.Builder()
                 .setBackpressureStrategy(ImageAnalysis.STRATEGY_KEEP_ONLY_LATEST)
-//                analysisBuilder.setTargetResolution(Size(1440, 1920))
+
+            val camProfile = CamcorderProfile.get(CamcorderProfile.QUALITY_HIGH)
```
Comment (navaronbracke) [inline android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt:192]: CamcorderProfile.get() is deprecated. See https://developer.android.com/reference/android/media/CamcorderProfile#get(int)

**Comment by EArminjon** [comment]
Up, if i'm not wrong image quality is important for ML Kit image recognition. Based on our test, by default the package use a resolution of 640x480 which lead to many false positive or many bad result during scans.

By upgrading the quality manually theses issue disappears...

**Comment by EArminjon** [comment]
@bswhite1 btw, no issue around aspect Ratio ?

**Comment by bswhite1** [comment]
@EArminjon the setTargetResolution call is definitely the thing that fixes it on the samsungs. I have been running my fork in production since May with no issues or complaints. I feel like I couldn't get the CamcorderProfile.getAll() to work for some reason, but it has been a while and I don't fully remember.

**Comment by EArminjon** [comment]
@bswhite1 with this PR i detect a weird aspect ratio :

371838172_134906163025928_4303201540377949977_n



___________________________________________________

Without this PR : 

371911406_6594867283943308_8245423499940002420_n

File: android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt (hunk 1 above)
Comment (EArminjon) [reply android/src/main/kotlin/dev/steenbakker/mobile_scanner/MobileScanner.kt:192]: It's deprecated and setTargetResolution is also deprecated.

**Comment by EArminjon** [comment]
@bswhite1  Can you tell me if https://github.com/juliansteenbakker/mobile_scanner/pull/760 fix this issue ?

**Comment by bswhite1** [comment]
@EArminjon Yes, #760 fixes the issue and it does have a better aspect ratio.

**Comment by EArminjon** [comment]
> @EArminjon Yes, #760 fixes the issue and it does have a better aspect ratio.

Okay ty for your reply ! To help maintainer to determine which PR to merge, maybe you close this one please ?

**Comment by bswhite1** [comment]
closing in favor of https://github.com/juliansteenbakker/mobile_scanner/pull/760

**Pull request closed without merging**